modules/learn/pages/data/transactions.adoc

```diff
@@ -5,38 +5,23 @@
 [abstract]
 {description}
 
-[#understanding-transactions]
-== Understanding Transactions
-
-_Distributed ACID Transactions_ are operations that ensure that when multiple documents need to be modified such that only the successful modification of all justifies the modification of any, either all the modifications do occur successfully; or none of them occurs.
-This _Atomicity_ supports _insert_, _update_, and _delete_ operations, across any number of documents.
-
-Transactions make use of the xref:learn:data/durability.adoc[Durability] provided by _synchronous writes_.
 
-During execution, for each document, a transaction maintains a modified copy of the document in the xref:learn:data/extended-attributes-fundamentals.adoc[Extended Attributes] area of the document's meta data.
-Consequently, the changes that occur during the transaction prior to commitment &#8212; these constituting _uncommitted_ (or _dirty_) data &#8212; are non-readable by any operation other than this transaction itself.
-Only following commit of the documents modified by the transaction &#8212; when the modified data-copy is removed from the meta data area, and is written over the main body of the document’s data &#8212; do the corresponding changes become readable by other transactions and operations.
-This isolation-level is known as https://jepsen.io/consistency/models/read-committed[Read Committed].
-Note that this use of a document’s extended attributes area results in an approximate doubling of its pre-transaction size, while the transaction is underway.
-In consequence, transactions can only be used on documents whose maximum size is 10 MB (which is half of the maximum-permitted document-size, 20 MB).
+Couchbase provides Distributed ACID Transactions for K-V (data) and Query operations -- these are available through APIs in 
+xref:cxx-txns::distributed-acid-transactions-from-the-sdk.adoc[C++], 
```

Comment:
The somewhat limited APIs are one of our weaker points currently, maybe shouldn't be first thing we stress?  E.g. for any service we don't mention the APIs in the first line.

Reply:
otoh we're promoting that Query is now covered.